### PR TITLE
Add mdm push notifications

### PR DESCRIPTION
**Pull request by signix**
Adding MDM Notificaiton type.
Let me know if any rework is necessary
Thank you

### CODE REVIEW

File: dotAPNS/ApplePush.cs [hunk 1]
```diff
@@ -387,7 +387,9 @@ public object GeneratePayload()
                     apsAsDict[customApsProperty.Key] = customApsProperty.Value;
             }
 
-            return payload;
+            var dict = (IDictionary<string, object>)payload;
```
Comment (alexalok) [inline dotAPNS/ApplePush.cs:390]: Wouldn't that remove the `aps` section for all push types instead of only mdm?

**Review by alexalok — CHANGES_REQUESTED** [review]
Hi @signix, thank you for your contribution! I have but a single remark for you to look over.

**Comment by AnisTigrini** [comment]
Hi there, could we re-open this branch for merging ? We would actually need it as a nugget package for one of our projects.
Thanks a lot @alexalok

File: dotAPNS/ApplePush.cs (hunk 1 above)
Comment (AnisTigrini) [reply dotAPNS/ApplePush.cs:390]: You could add something like :

```
var dict = (IDictionary<string, object>)payload;
if (Type == ApplePushType.Mdm)
    dict.Remove("aps");
 return dict;
```

**Comment by alexalok** [comment]
superseded by #125

**Comment by signix** [comment]
Thank you @alexalok

**Pull request closed without merging**